packages/lex-cli/src/util.ts

```diff
@@ -20,17 +26,33 @@ export function readAllLexicons(paths: string[]): LexiconDoc[] {
   return docs
 }
 
+const parseJSON = (
+  path: string,
+  str: string
+): Record<string, unknown> | Record<string, unknown>[] => {
+  if (path.endsWith('.json')) {
+    return JSON.parse(str)
+  }
+  if (path.endsWith('.jsonc')) {
+    return jsonc.parse(str)
+  }
+  if (path.endsWith('.jsonl')) {
+    return jsonl.parse(str)
+  }
```

Comment:
TODO: need to pick parsers, does the org have any licensing requirements for dependencies?